task01.rst

```diff
@@ -0,0 +1,97 @@
+###################
+Why I love Python
+###################
+
+Why I love **Python**, a language worth knowing.
+
+**********************
```

Comment:
Heading over/underlines should be the same length as the text they're over/underlining eg:

```
#####
Hello
#####
```
Not
```
############
Hello
############
```